ethereum_history_api/circuits/lib/src/storage.nr

```diff
@@ -13,3 +15,18 @@ struct StorageProof {
     proof: [u8; STORAGE_PROOF_LEN],
     depth: Field
 }
+
+struct StorageProofsWithStorageRoot<N> {
+    storage_root: Bytes32,
+    proofs: [StorageProof; N]
+}
+
+fn verify_storage_values<N>(storage_root: Bytes32, proofs: [StorageProof; N]) {
+    for i in 0..N {
+        verify_storage_value(storage_root, proofs[i]);
+    }
+}
+pub(crate) fn verify_storage_value(storage_root: Bytes32, proof: StorageProof) {
+    let trie_proof = TrieProof { key: proof.key, value: proof.value, proof: proof.proof, depth: proof.depth };
+    assert(trie_proof.verify_storage_root(storage_root), "TrieProof: Invalid storage root");
```

Comment:
`verify_storage_root` returns `bool` but in fact it never returns false but will always return true or fails with assertion

Reply:
Perhaps a comment?

Reply:
Added comment

Reply:
One extra dot at the end of the comment;)